src/classes/ACCT_CascadeDeleteLookups_TDTM.cls-meta.xml

```diff
@@ -3,7 +3,7 @@
     <apiVersion>37.0</apiVersion>
     <packageVersions>
         <majorNumber>3</majorNumber>
-        <minorNumber>12</minorNumber>
+        <minorNumber>13</minorNumber>
```

Comment:
@lparrott @pedrodalcol  I believe all of these API version updates will go away if this merge conflict PR is resolved: https://github.com/SalesforceFoundation/Cumulus/pull/3551